force-app/main/default/classes/UTIL_Describe.cls

```diff
@@ -799,6 +799,10 @@ public class UTIL_Describe {
         return objectName;
     }
 
+    public Boolean isFieldSortable(Schema.SObjectField field) {
+        return field.getDescribe().isSortable();
```

Comment:
I assume this takes into account the encryption status of the field/org? I don't see that referenced anywhere in the docs.

Reply:
It does, if the field is encrypted then it won't be sortable. Other fields can return false for this always too, like Long or Rich Text.